Sources/message/ALMessage.m

```diff
@@ -296,20 +296,16 @@ - (NSString *)getVOIPMessageText {
 }
 
 - (BOOL)isMsgHidden {
-    BOOL hide = [[self.metadata objectForKey:@"hide"] boolValue];
-
-    // Check messages that we need to hide
-    NSArray *keys = [ALApplozicSettings metadataKeysToHideMessages];
     
-    if (keys != nil) {
-        for (NSString *key in keys) {
-            // If this key is present then it's a hidden message
-            if ([self.metadata objectForKey:key] != nil) {
-                return true;
-            }
-        }
+    if (self.message && ([self.metadata objectForKey:@"KM_ASSIGN_TO"] != nil || [self.metadata objectForKey:@"KM_ASSIGN_TEAM"] != nil)) {
+        return false;
+    }
+    
+    if ([self.metadata objectForKey:@"KM_ASSIGN"] != nil) {
```

Comment:
These new checks will affect existing users as well, make sure that's fine.

Reply:
it won't affect existing users..since we are not processing hidden message to db